app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEffectHandlerTest.kt

```diff
@@ -162,6 +162,22 @@ class BloodSugarEntryEffectHandlerTest {
     verifyNoMoreInteractions(ui)
   }
 
+  @Test
+  fun `set specific date on input fields and date button when prefill date for update effect is received`() {
+    // given
+    val entryDate = LocalDate.of(2020, 2, 14)
+    userClock.setDate(LocalDate.of(2020, 2, 14))
+
+    // when
+    testCase.dispatch(PrefillDate.forUpdateEntry(userClock.instant()))
```

Comment:
We generally use `Instant.now(clock)` instead of `clock.instant()` throughout the codebase. Could we follow this convention here as well, please?